pkg/testutil/fakes.go

```diff
@@ -461,6 +462,33 @@ func (s simpleServiceNamespaceLister) Get(name string) (*v1.Service, error) {
 	return s.kubeClient.CoreV1().Services(s.namespace).Get(context.TODO(), name, metav1.GetOptions{})
 }
 
+// simple ClusterRoleLister implementation
```

Comment:
the first word after the `//` is meant to be the name of the thing the comment is about (godoc)

Reply:
got it.  I think these are cleaned up now.